diff -r e4b1aba80949 cmake_modules/FindCOLLADA_DOM.cmake
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/cmake_modules/FindCOLLADA_DOM.cmake	Tue Aug 14 15:26:16 2012 +0900
@@ -0,0 +1,35 @@
+
+# @author Shin'ichiro Nakaoka
+
+
+if(NOT COLLADA_DOM_DIR)
+    if(UNIX)
+      find_path(
+        COLLADA_DOM_DIR
+        NAMES include/collada-dom/dom.h
+        PATHS /usr/local /usr
+        DOC "the top directory of collada")
+     endif()
+endif()
+
+if(COLLADA_DOM_DIR)
+  string(REGEX REPLACE "/$" "" COLLADA_DOM_DIR ${COLLADA_DOM_DIR})
+  if(EXISTS ${COLLADA_DOM_DIR}/include/dom.h)
+    set(COLLADA_DOM_FOUND TRUE)
+  endif()
+endif()
+
+set(COLLADA_DOM_DIR ${COLLADA_DOM_DIR} CACHE PATH "The top directory of collada")
+set(COLLADA_DOM_INCLUDE_DIRS ${COLLADA_DOM_DIR}/include ${COLLADA_DOM_DIR}/include/1.5)
+set(COLLADA_DOM_LIBRARY_DIRS ${COLLADA_DOM_DIR}/lib )
+set(COLLADA_DOM_LIBRARIES collada15dom)
+
+if(COLLADA_DOM_FOUND)
+  if(NOT Collada_FIND_QUIETLY)
+    message(STATUS "Found collada headers in ${COLLADA_DOM_INCLUDE_DIR}")
+   endif()
+else()
+  if(Collada_FIND_REQUIRED)
+    message(FATAL_ERROR "Could not find collada. Please set COLLADA_DOM_DIR correctly")
+  endif()
+endif()
