Index: server/ControllerBridge/CMakeLists.txt
===================================================================
--- server/ControllerBridge/CMakeLists.txt	(revision 2372)
+++ server/ControllerBridge/CMakeLists.txt	(revision 2373)
@@ -19,10 +19,9 @@
     ${program}
     hrpCorbaStubSkel-${OPENHRP_LIBRARY_VERSION}
     ${OPENRTM_LIBRARIES}
-    boost_filesystem-mt boost_program_options-mt boost_regex-mt)
-  if(APPLE)
-    target_link_libraries(${program} boost_system-mt)
-  endif() 
+    boost_filesystem-mt boost_program_options-mt boost_regex-mt
+    boost_system-mt
+    )
 elseif(WIN32)
   add_definitions(${OPENRTM_DEFINITIONS})
   set_target_properties(${program} PROPERTIES DEBUG_POSTFIX d )
